ElectronicObserver/Window/Tools/EquipmentList/EquipmentListViewModel.cs

```diff
@@ -28,13 +27,12 @@ public partial class EquipmentListViewModel : WindowViewModelBase
 		Filter = "CSV|*.csv|File|*",
 	};
 
-	public List<ColumnProperties> EquipmentGridColumnProperties { get; set; } = new();
-	public List<SortDescription> EquipmentGridSortDescriptions { get; set; } = new();
+	public DataGridViewModel EquipmentGridViewModel { get; set; } = new();
 	public GridLength EquipmentGridWidth { get; set; } = GridLength.Auto;
 
 	// todo: doesn't seem to work in the current implementation
-	public List<ColumnProperties> DetailGridColumnProperties { get; set; } = new();
-	public List<SortDescription> DetailGridSortDescriptions { get; set; } = new();
+	// todo: check if new implementation is working
+	public DataGridViewModel EquipmentDetaiGridViewModel { get; set; } = new();
```

Comment:
Typo in EquipmentDetailGridViewModel and no, it doesn't work. Select an equipment with multiple detail items, sort by something, select a different equipment, sort data is lost.